packages/intersection-observer/README.md

```diff
@@ -104,4 +105,8 @@ Updated to Solid 1.3
 
 Minor improvements
 
+1.3.0
```

Comment:
Breaking changes should probably have a major release.

Reply:
Yup, I'll bump everything when I'm confident it's ready. Sorry just threw this together really fast.